src/rc_allocator.c

```diff
@@ -848,7 +859,12 @@ rc_allocator_print_allocated(rc_allocator *al)
          platform_default_log("%8lu %12lu     %u\n", i, ext_addr, ref);
       }
    }
-   platform_default_log("%sFound %lu extents with allocated pages.\n",
+
+   char   bytes_to_str[SIZE_TO_STR_LEN];
+   uint64 nbytes = (found * al->cfg->io_cfg->extent_size);
+   size_to_str(bytes_to_str, sizeof(bytes_to_str), nbytes);
+   platform_default_log("%sFound %lu extents (%s) with allocated pages.\n",
                         (print_curly ? "}\n" : ""),
-                        found);
+                        found,
+                        bytes_to_str);
```

Comment:
New output from this change:

`Found 10495 extents (~1.28 GiB) with allocated pages.`